py/pyconstants.cpp

```diff
@@ -56,6 +56,7 @@ void init_constants(py::module& torchcraft) {
   setEnumDict<BW::UserCommandType>(constants, "usercommandtypes");
   setEnumDict<BW::Order>(constants, "orders");
   setEnumDict<BW::TechType>(constants, "techtypes");
+  setEnumDict<BW::UpgradeType>(constants, "upgradetypes");
```

Comment:
I think your referring to the state values, that's in py/pyreplayer.cpp. These are the constants/enums.